### DISCUSSION TITLE
Issue Report with Special External Display Settings

### DISCUSSION THREAD

**Discussion by basilkorompilias** [emoacht/Monitorian#525] [General]
Hey there and thank you for your banging app! :)
I had a problem recently where it wouldn't work for some time and made me kind of desperate. Tried other apps but there is no better option! 

Long story short, I saw you write that some setting in the display itself might not work (something about a DCC, which my display doesn't have, anyway). So I played a bit with the settings to see, and I found out that when there is a custom setup with the colours from the display, it doesn't work. Specifically I had colours set as cold, and I changed it to "Custom" to make it work.

To recap. It may be helpful to add to your readme that for some displays it won't work unless you have the "Custom" option selected, without the need to actually configure it. You can just say, if you are desperate, try change some display settings from the display itself :P  

Finally brightness works again on my external monitor! :D 
I can sleep at night with Venice Live playing on the lowest brightness...
Thanks for that <3

https://www.youtube.com/watch?v=K_Vg94nBiaY&list=PLUGJu5jJ9Ma32GJzBdI3glxEzXg1k5qDZ&index=7

**Comment by emoacht** [comment]
Thank you for the information.
Could you provide the vendor and model of that monitor? Also, the photo of the settings like the following will be helpful.
https://github.com/emoacht/Monitorian/blob/master/Images/Dell_ddcci.jpg

**Comment by basilkorompilias** [comment]
I can't screenshot, so I attach a photo. It is the User Color temperature that needs to be selected. It was in Greek before and it said custom. The brand is a Greek one (Chinese rebrand probably) Turbo X 21.5' TS2142RBB. Nothing fancy, a 1080p. I have it connected through HDMI with my windows 11 laptop (9 years old).

Cheers
![IMG_20231006_234816](https://github.com/emoacht/Monitorian/assets/23310772/0a5fb957-436a-48b3-8a74-0b9eb436ec7b)

**Comment by emoacht** [comment]
Thank you. But where is "Custom"?

**Comment by basilkorompilias** [comment]
Occasionally they translate things differently in Greek. They did translate the word "User" to "Προσαρμογή" which means customisation. I had the menu in Greek before. I changed it to English for you. 

Well, the word "user" is afterall a negative one as it denotes addiction, so for once good for them 😂. You don't need to deal with so many details. Just for your reference. Cheers
